site/developer/model-documentation/generate-model-documentation.qmd

```diff
@@ -1,128 +1,77 @@
 ---
 title: "Generate model documentation"
```

Comment:
This topic should also be renamed since it covers much more than just the _generation_ of documentation. This is effectively the more hands-on end-to-end workflow for documenting a model. 

```suggestion
title: "Document models"
```

Reply:
Should I change the filename as well?